Gom4ziz/Source/Presenter/UserViewModel.swift

```diff
@@ -16,10 +16,17 @@ final class UserViewModel {
     private let addUserUsecase: AddUserUsecase
     private let disposeBag: DisposeBag = .init()
     let userObservable: BehaviorRelay<Loadable<User>> = .init(value: .notRequested)
+    var user: User?
```

Comment:
userObservable에서 user를 꺼내기 번거롭다면, 계산 프로퍼티로 구현하는게 어떨까요
```suggestion
    var user: User? {
      get {
      userObservable.value.value
      }
    }
```